### PR TITLE
Add analysis page

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR adds a basic analysis page for interacting with Analysis Services.

### CODE REVIEW

File: static/js/components/AnalysisServicePage.jsx [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+      // update
+
+      await dispatch(analysisServicesActions.fetchAnalysisServices());
+    };
+
+    getAnalysisServices();
+
+    // Don't want to reset everytime the component rerenders and
+    // the defaultStartDate is updated, so ignore ESLint here
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (guynir42) [inline static/js/components/AnalysisServicePage.jsx:159]: Is it weird that I pay more attention to the code comments than the code when I'm reviewing?

File: config.yaml.defaults [hunk 2]
```diff
@@ -195,6 +197,10 @@ app:
           icon: TornadoOutlined
           url: /galaxies
 
+        - name: Analysis Services
+          icon: SentimentSatisfied
```
[the remark this answers is not in the document]
Comment (mcoughlin) [reply config.yaml.defaults:201]: This icon made.me so happy when I found it.

File: static/js/components/AnalysisServicePage.jsx (hunk 1 above)
Comment (mcoughlin) [reply static/js/components/AnalysisServicePage.jsx]: removing :)

File: static/js/ducks/analysis_services.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import messageHandler from "baselayer/MessageHandler";
+
+import * as API from "../API";
+import store from "../store";
+
+const FETCH_ANALYSIS_SERVICES = "skyportal/FETCH_ANALYSIS_SERVICES";
+const FETCH_ANALYSIS_SERVICES_OK = "skyportal/FETCH_ANALYSIS_SERVICES_OK";
```
[the remark this answers is not in the document]
Comment (mcoughlin) [reply static/js/ducks/analysis_services.js:7]: I am cool with either way. We have a lot of these plurals, but I doubt anyone is particularly keen on the current way of doing it. Let's discuss at the meeting today.

File: static/js/components/NewAnalysisService.jsx [hunk 4]
```diff
@@ -9,8 +9,10 @@ import dayjs from "dayjs";
 import utc from "dayjs/plugin/utc";
 
 import GroupShareSelect from "./GroupShareSelect";
-import { submitAnalysisService } from "../ducks/analysis_service";
-import { fetchAnalysisServices } from "../ducks/analysis_services";
+import {
+  submitAnalysisService,
+  fetchAnalysisServices,
+} from "../ducks/analysis_services";
```
Comment (guynir42) [inline static/js/components/NewAnalysisService.jsx:15]: To me this looks a lot more reasonable than what we had before...

**Review by guynir42 — APPROVED** [review]
Looks good.

**Comment by mcoughlin** [comment]
Thanks @guynir42

**Pull request merged**